libs/platform/src/lib/components/menu/menu.module.ts

```diff
@@ -0,0 +1,13 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { MenuComponent } from './menu-component/menu.component';
+import { MenuItemComponent } from './menu-component/menu-item.component';
+import { FundamentalNgxCoreModule, MenuKeyboardService } from '@fundamental-ngx/core';
+
+@NgModule({
+    declarations: [MenuComponent, MenuItemComponent],
+    imports: [CommonModule, FundamentalNgxCoreModule],
+    exports: [MenuComponent, MenuItemComponent],
+    providers: [MenuKeyboardService]
+})
+export class MenuModule {}
```

Comment:
There's already a `MenuModule` in core. Will this cause name conflicts?

Reply:
It could, yes, if one is not careful while importing. Would you rather I rename it to PlatformMenuModule instead like it is done for Search and Button?

Reply:
Again we dont need to create some extra names as Menu is straight forward. If you want to use both of the for some reason you can import it like this:

```
import {MenuComponnet as PlatformMenu} from 'xxxx'
```

As we keep working on more components we will have similar names but it does not mean its bad , when we work on the e.g. dropdown, we want to still keep it a dropdown.

Reply:
What happens when someone imports both "core" and "platform" libraries in their module?
```
@NgModule({
  declarations: [ ... ],
  imports: [
    ...
    FundamentalNgxCoreModule,
    FundamentalNgxPlatformModule,
    ...
  ],
  providers: [ ... ],
})
```

Will there be name conflicts in this case, because both libraries have imports with the same names?